class.jetpack-gutenberg.php

```diff
@@ -25,6 +36,8 @@ function jetpack_register_block( $slug, $args = array() ) {
 /**
  * Helper function to register a Jetpack Gutenberg plugin
  *
+ * @deprecated 7.0.0 Use jetpack_activate_gutenberg_extension() instead
```

Comment:
Did you mean to say "Use jetpack_register_gutenberg_extension()", since there is no "jetpack_activate_gutenberg_extension()"?